spec/unit/rust-crypto/rust-crypto.spec.ts

```diff
@@ -1567,6 +1567,56 @@ describe("RustCrypto", () => {
         });
     });
 
+    describe("withdraw verification", () => {
+        let rustCrypto: RustCrypto;
+        let olmMachine: Mocked<RustSdkCryptoJs.OlmMachine>;
+
+        beforeEach(() => {
```

Comment:
I won't insist on it, but I have "opinions" on this style: [[setUp and tearDown considered harmful](https://artificialworlds.net/blog/2013/03/27/setup-and-teardown-considered-harmful/)](https://artificialworlds.net/blog/2013/03/27/setup-and-teardown-considered-harmful/).

I personally would prefer a `createRustCrypto` function that is called at the beginning of each test. I feel it makes tests easier to read, and removes a risk of cross-test contamination.

Reply:
Done